cmd/prober/prober.go

```diff
@@ -125,11 +130,17 @@ func init() {
 	flag.StringVar(&addr, "addr", ":8080", "Port to expose prometheus to")
 
 	flag.StringVar(&rekorURL, "rekor-url", "https://rekor.sigstore.dev", "Set to the Rekor URL to run probers against")
+	// TODO: use the production instance
+	flag.StringVar(&rekorV2URL, "rekor-v2-url", "https://log2025-alpha1.rekor.sigstage.dev", "Set to the Rekor V2 URL to run probers against")
 	flag.StringVar(&fulcioURL, "fulcio-url", "https://fulcio.sigstore.dev", "Set to the Fulcio URL to run probers against")
 	flag.StringVar(&fulcioGrpcURL, "fulcio-grpc-url", "fulcio.sigstore.dev", "Set to the Fulcio GRPC URL to run probers against")
 
 	flag.StringVar(&tsaURL, "tsa-url", "https://timestamp.sigstore.dev", "Set to the TSA URL to run probers against")
 
+	// TODO: use the production mirror
+	flag.StringVar(&tufRepoURL, "tuf-repo-url", tuf.StagingMirror, "Set to the TUF repository URL to run probers against")
```

Comment:
It's a little unintuitive to me to be using both `--rekor-v2-url` and TUF to discover URLs, because the shard name is in the URL. Whatever is in `--rekor-v2-url` should also already be in TUF, I'm not sure there's a case when you'd want to check another URL?

Reply:
This will be useful to run the prober against a local or development instance, or any other instance that we haven't already included in a TUF repo.